specs/core/1_custody-game.md

```diff
@@ -468,13 +479,13 @@ def process_chunk_challenge(state: BeaconState,
             record.chunk_index != challenge.chunk_index
         )
     # Verify depth
-    depth = math.log2(next_power_of_two(get_custody_chunk_count(challenge.attestation.data.crosslink)))
+    depth = math.ceil(math.log2(get_custody_chunk_count(challenge.attestation.data.crosslink)))
```

Comment:
I am not sure this is a particularly nice way of doing things, but I couldn't find this function (I thought it was in SSZ somewhere) and because we already have `math` imported, I did this.

Reply:
I just found a nice way to do this in python without relying on floating point arithmetic: Use the bit_length method of an integer:
```python
def ceillog2(x):
    return x.bit_length()
```

I suggest we use this as it would not add dependencies on FP math that can potentially be flaky in extreme conditions?